fortuna/src/config.rs

```diff
@@ -115,6 +118,9 @@ pub struct EthereumConfig {
     /// Address of a Pyth Randomness contract to interact with.
     pub contract_addr: Address,
 
+    /// How many blocks to wait before revealing the random number.
+    pub confirmation_blocks: BlockNumber,
```

Comment:
Very very minor nitpick and possibly bikeshedding, but I think the name is a bit confusing, `reveal_delay_blocks` might be a bit better.